Source/WebCore/platform/graphics/angle/GraphicsContextGLANGLE.cpp

```diff
@@ -2959,6 +2959,96 @@ void GraphicsContextGLANGLE::drawBuffersEXT(GCGLSpan<const GCGLenum> bufs)
     GL_DrawBuffersEXT(bufs.size(), bufs.data());
 }
 
+PlatformGLObject GraphicsContextGLANGLE::createQueryEXT()
+{
+    if (!makeContextCurrent())
+        return 0;
+
+    GLuint name;
```

Comment:
Could we get the out variables all initialized?

Reply:
Done.